### DISCUSSION TITLE
Problem when executing conformance for 3 models together

### DISCUSSION THREAD

**Discussion by LukeshSingla** [onvif/testspecs#41] [Q&A] [answered]
Dear Team,
We are doing conformance of our client, but while running the conformance test against 3 cameras we found Motion Alarm testcase under Profile T is failing for Bosch_Autodome, but if we run the test for same camera as standalone (with capture only from Bosch_Autodome camera), Motion Alarm testcase is working fine. So we suspect there is some bug in conformance tool. We are using latest tool version 22.12
![image](https://user-images.githubusercontent.com/94896881/214555061-973ddd75-381c-4300-98fb-2cfdd81aae4f.png)
<img width="960" alt="Bosch_Motion alarm success" src="https://user-images.githubusercontent.com/94896881/214555128-81067d93-ddbf-41e5-90a7-3290eae8f4d7.png">

**Comment by LukeshSingla** [comment]
![Bosch_Motion alarm failed](https://user-images.githubusercontent.com/94896881/214555514-300da69d-b292-48e7-9c22-139721b376b1.png)
When done 3 cameras together

**Comment by madhurao68** [comment, marked as answer]
Our test tool vendor investigated and would need traces to analyze the issue. Appreciate if you could share the screenshot of the 1st page with list of client and devices since it looks like it is a mix of client and devices.

![pic](https://user-images.githubusercontent.com/73811069/228236704-d32f9dd6-81ce-4906-a426-94109e5f18af.png)
